queries.sql

```diff
@@ -9,3 +9,28 @@ SELECT  * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
 
+/* Database Transactions */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+COMMIT
+
```

Comment:
- [ ] @Nejjari001 kindly add a screenshot showcasing the change before the commit was made and after confirming the change was sustained after the commit